src/main/java/net/openhft/chronicle/threads/MediumEventLoop.java

```diff
@@ -485,7 +490,6 @@ protected void addNewHandler(@NotNull final EventHandler handler) {
                     mediumHandlers.add(handler);
                     updateMediumHandlersArray();
                     handler.eventLoop(parent != null ? parent : this);
-                    handler.loopStarted();
```

Comment:
This was being called when addHandler() was called prior to the event loop being started. It's actually called later and properly guarded to only run when the event loop is started
See https://github.com/OpenHFT/Chronicle-Threads/pull/158/files#diff-73b32a0226a3cea0afab5d0b363b1b401ae5b69cc707c504bd79058cd5f267ecR509

Reply:
Maybe improve net.openhft.chronicle.threads.EventGroupTest.TestHandler#loopStarted to check that only called once